src/routes/api/v1/algolia/+server.ts

```diff
@@ -4,5 +4,6 @@ import { loadPageMetadataMap } from "$lib/loadPageMetadataMap";
 
 export const GET = (async () => {
   const { pageMetadataMap } = await loadPageMetadataMap({ includeBreadcrumbs: false });
-  return json(Array.from(pageMetadataMap.values()));
+  // exclude the home page and pages where we can't determine the URL
+  return json(Array.from(pageMetadataMap.values()).filter((page) => page.url && !page.isRoot));
```

Comment:
note that right now, this in isolation doesn't filter out these results until we update the Algolia index directly. Uploading this entire response as a fresh set of records into Algolia _would_ remove them.

I'm working on a POST method to compliment this that would update things as needed